// Copyright (c) 2012, Cloudera, inc.

package kudu;

enum DataType {
  UINT32 = 0;
  INT32 = 1;
  STRING = 3;
}
